code-studio/package.json

```diff
@@ -67,6 +68,7 @@
     "mochify": "^2.15.0",
     "mochify-istanbul": "^2.4.1",
     "node-sass": "^3.4.2",
+    "node-watch": "^0.3.5",
```

Comment:
Did we end up using node-watch?  If not, let's remove this line from the dependency list.